ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -910,7 +910,7 @@ Loader {
 
                 quickActions: [
                     Loader {
-                        active: d.addReactionAllowed && delegate.hovered && !delegate.hideQuickActions
+                        active: d.addReactionAllowed && delegate.hovered
```

Comment:
hmm, why don't we want to keep `&& !delegate.hideQuickActions`? 🤔

Reply:
I didn't want to tamper too much with the QML code because it's not my jurisdiction, but here it's because we want to still show the Emoji reacitons when doing a right-click on a message in a read-only channel.

The code is not really made for that, so that's why it's a bit weird in places.

Reply:
Looks like it's important to keep here.
Guessing from  the definition:
```qml
readonly property bool hideQuickActions: root.isChatBlocked ||
                  root.placeholderMessage ||
                  root.isInPinnedPopup ||
                  root.editModeOn ||
                  !root.rootStore.mainModuleInst.activeSection.joined
```

we should hide the _and disable the loaders_ in a few cases. 
`isInPinnedPopup` is also covered by `addReactionAllowed`. But all other cases doesn't seem to be covered.

I understand that `isChatBlocked` doesn't consider `canPostReactions` at all. 
So we should probably review the set of properties here so that it fit our needs.